jekyll/_docs/faq.md

```diff
@@ -6,40 +6,61 @@ categories: [getting-started]
+
+You can **subscribe by email, RSS or JSON**. **[Follow these instructions](https://discuss.circleci.com/t/how-to-subscribe-to-announcements-and-notifications-from-circleci-email-rss-json/5616) to subscribe now.**
+
+You may also like to follow our **[changelog](https://circleci.com/changelog/)** to see what has been released.
+
+Our **[status page](http://status.circleci.com/)** gives information on current system status, outages and past incidents.
+
 ## Authentication
 
 ### Do you support BitBucket or GitLab?
-Currently we support authentication with GitHub and Bitbucket. We do look at feature requests on [Discuss](https://discuss.circleci.com/) if you want to show support for other version control systems.
+
+Currently we support authentication with GitHub and Bitbucket. Please vote on relevant [Feature REquest](https://discuss.circleci.com/c/feature-request) if you want to show support for other version control systems.
 
 ### I can’t give CircleCI the access to all my private repositories. What do I do?
+
+Please see our guides and explanations about [GitHub and Privacy](https://circleci.com/docs/github-privacy/)
+
 GitHub has only recently added the fine-grained permissions options, and
 we are still working on supporting them.
 
-In the meantime, the suggested workaround is to create an additional
-user on GitHub with a limited set of permissions and use that account to
+In the meantime, the [suggested workaround](https://circleci.com/docs/github-security-ssh-keys/) is to create an additional user on GitHub with a limited set of permissions and use that account to
 perform the builds on CircleCI.
 
 ### I updated my email address on GitHub, and it does not show up on CircleCI
+
 We refresh GitHub information once a day to stay within GitHub’s API
 limits, so check your profile page later – it will be right there.
 
 ## Billing & Plans
+
 ### Can I build more than one project if I only have one container?
+
 Absolutely. In this case the builds will run one at a time, one after
 another.
 
 ### How do I stop CircleCI from building a project?
+
 If you get everyone who follows the project on CircleCI to unfollow it, we
 will automatically stop building it.
 
 ## Integrations
-### Can I send HipChat / Slack / IRC notifications for specific branches only?
-We don’t currently offer this kind of selective notifications, but the
-functionality is in the works. Keep an eye on our
-[changelog](https://circleci.com/changelog/) to be notified as soon as
-this feature is available.
+
+### Can I get notifications to team chat applications?
+
+CircleCI supports Slack, HipChat, Campfire, Flowdock and IRC notifications; you configure these notifications from your project’s **Project Settings > Notifications** page.
+
+### Can I send Slack / HipChat / IRC notifications for specific branches only?
+
+There is [experimental support for per branch chat notifications](https://circleci.com/docs/configuration/#per-branch-notifications). 
```

Comment:
[experimental support for per branch chat notifications](https://circleci.com/docs/configuration/#per-branch-notifications) => [experimental support for per branch chat notifications]({{site.baseurl}}/configuration/#per-branch-notifications)